components/server/src/user/user-service.spec.db.ts

```diff
@@ -4,51 +4,44 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-import { DBUser, TypeORM, UserDB } from "@gitpod/gitpod-db/lib";
+import { Experiments } from "@gitpod/gitpod-protocol/lib/experiments/configcat-server";
```

Comment:
this just migrates the single existing user-service test to the new infrastructure and prepares this module to get more test for further user services methods including auth check testing as done in prganization-service.spec.db.ts and project-service.spec.db.ts.